src/validators/ClassValidator.ts

```diff
@@ -376,7 +393,7 @@ export class BsClassValidator {
                         relatedInformation: [{
                             location: util.createLocation(
                                 URI.file(alreadyDefinedClass.file.srcPath).toString(),
-                                this.classes.get(lowerName).range
+                                alreadyDefinedClass.range
```

Comment:
It was actually to satisfy the checker, since frustratingly enough TS doesn't understand this:

```typescript
// map has type Map<string, number>
if (map.has("foo")) {
    const n: number = map.get("foo")
    bar(n);
}
```
Because `has` doesn't refine the return value of `get`, TS thinks `n` can be `undefined`. My usual refactoring is

```typescript
const n: number | undefined = map.get("foo");
if (n !== undefined) {
    bar(n);
}
```

Fortunately 3/4 of this refactoring was already done, so I just grabbed the variable that already existed.

Can't hurt to be more efficient though 😁